config/Dockerfiles/tests.d/aws/08_aws_vpc_internet_gateway

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+
+# Verify the openstack router provisioning
+# distros.exclude: none
+# providers.include: aws
+# providers.exclude: none
+
+DISTRO=${1}
+PROVIDER=${2}
+TARGET0="aws-ec2-vpc-net-test"
+TARGET1="aws-ec2-vpc-internet-gateway"
+pushd docs/source/examples/workspaces/${PROVIDER}
+function clean_up {
+    set +e
+    linchpin -w . -vvv --template-data '{ "vpc_id": '"${VPC_ID}"' }' destroy "${TARGET1}"
+    linchpin -w . -vvv --template-data '{ "distro": '${DISTRO}""'}' destroy "${TARGET0}"
+    rm -rf resources/
+}
+
+trap clean_up EXIT
+
+linchpin -w . -vvv --template-data '{ "distro": '${DISTRO}""'}' up "${TARGET0}"
+cat ./resources/ec2vpc.output
+VPC_ID=$(awk -F '"' '{print $48}' ./resources/ec2vpc.output)
```

Comment:
This should be builtin functionality, see how I did it:
https://github.com/CentOS-PaaS-SIG/linchpin/pull/1243